OverlayPlugin.Core/ClipboardTechSupport.cs

```diff
@@ -106,7 +109,12 @@ public ClipboardTechSupport(TinyIoCContainer container)
                 settings.Add(new List<string> { "Machina Region", repository.GetMachinaRegion().ToString() });
                 string gameVersion = repository.GetGameVersion();
                 settings.Add(new List<string> { "Game Version", gameVersion != "" ? gameVersion : "(not running)" });
-                settings.Add(new List<string> { "Screen Mode", GetFFXIVScreenMode(repository.GetCurrentFFXIVProcess()) });
+
+                if (screenMode == null)
+                {
+                    repository.RegisterProcessChangedHandler(GetFFXIVScreenMode);
```

Comment:
It might make sense to re-apply the `"(unknown)"` default value here before registering the process handler, to perfectly replicate the previous functionality?

Otherwise this looks good now, I'll pre-approve.